### PR TITLE
support LRSchedule for pir.

### PR DESCRIPTION
**Pull request by winter-wang**
### PR Category

Auto Parallel

### PR Types

 Bug fixes

### Description

- support LRSchedule for pir program.

### Other
Pcard-67164

### CODE REVIEW

**Review by JZ-LIANG — DISMISSED** [review]
LGTM

File: paddle/fluid/pybind/pir.cc [hunk 1]
```diff
@@ -281,6 +297,45 @@ std::string GetValueName(Value value) {
...
+  if (p_mapper == nullptr) {
+    p_mapper = &mapper;
+  }
+  auto src_obj = py::cast(self);
+  auto new_obj = py::cast(self.Clone(*p_mapper));
+  for (auto item : src_obj.attr("__dict__").cast<py::dict>()) {
+    new_obj.attr(item.first.cast<std::string>().c_str()) = item.second;
+  }
+  return new_obj;
+}
```
Comment (zhiqiu) [inline paddle/fluid/pybind/pir.cc:311]: what about the calling original clone in c++ side?

**Review by zhiqiu — APPROVED** [review]
LGTM

File: paddle/fluid/pybind/pir.cc (hunk 1 above)
Comment (winter-wang) [reply paddle/fluid/pybind/pir.cc:311]: It will only clone the computation graph, some python properties such as "lr_rate" may be lost in the clone.

**Pull request closed without merging**